packages/common/scss/leaders.scss

```diff
@@ -0,0 +1,60 @@
+.leaders {
+  &__items {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    padding-left: 0;
+    list-style-type: none;
+    @include media-breakpoint-down($theme-responsive-text-breakpoint) {
+      display: initial;
+      width: 100%;
+    }
+  }
+
+  &__item {
+    .btn-link:hover,
+    .btn-link:active,
+    .btn-link:focus {
+      text-decoration: none;
+    }
+  }
+}
+
+.node-list {
+  $self: &;
+  &--centered {
+    #{ $self } {
+      &__node,
+      &__header,
+      &__footer,
+      &__body {
+        text-align: center;
+      }
+    }
+  }
+  &--header-secondary {
```

Comment:
Remove `--header-*` mods...
You could add a `--header-leaders` with a `$common-leaders-header-bg-color` var though...